### ISSUE TITLE
peerDependencies

### ISSUE THREAD

**Issue by varna** [panter/vue-i18next#92]
Hi,
I just installed latest version of `vue-i18next`, but it seems that you got some version mismatches?
I can see that `@panter/vue-i18next/typings/index.d.ts` and `i18next` have different names for some functions. In `package.json` I noticed:
```
  "peerDependencies": {
    "i18next": ">= 6.0.1",
    "vue": ">=2.0.0"
  },
```
Shouldn't you use `^` operator to target *minor* versions instead of `>=` which I never use, but I assume that targets any latest *major* versions :D

**Comment by varna** [comment]
I just started downgrading my i18next major versions one by one until i18next@^13
Now it works properly.

**Comment by albertpratomo** [comment]
Yes, I'm also having the same issue. Could you support the latest `i18next` (`^19`)?

**Comment by ram4nd** [comment]
I would also like to see i18next version update. Perhaps this needs an active fork.
